files/KoreBuild/scripts/dotnet-install.sh

```diff
@@ -959,31 +959,49 @@ do
     shift
 done
 
-if [ "$no_cdn" = true ]; then
-    azure_feed="$uncached_feed"
+local lockFile=""
+local waitTime=0
+
+while [ ! -f $lockFile ] && [ $waitTime < 120 ]
+do
+    sleep 10
+    waitTime += 10
+done
+
+if [ $waitTime > 120]; then
```

Comment:
Also, this is a common pitfall of bash. Use `$waitTime -gt 120`. Likewise above, use `-lt`. See  http://tldp.org/LDP/abs/html/comparison-ops.html

I think this is a syntax error. You need a space between 120 and ].